src/OSPSuite.R/MinimalImplementations/RQuantityPathToQuantityDisplayPathMapper.cs

```diff
@@ -19,9 +19,22 @@ protected override IEnumerable<PathElementId> DefaultPathElementsToUse(bool addS
 
          yield return PathElementId.Molecule;
 
+         //Container is defined? no need to use TopContainer
```

Comment:
seems to me that the comment belongs into `shouldDisplayTopContainer `(line 33)

Reply:
I'll remove it in another PR. Right now the package is released